packages/eslint-plugin/docs/rules/no-require-imports.md

```diff
@@ -50,6 +50,30 @@ console.log(require('../data.json').version);
 console.log(require('../package.json').version);
 ```
 
+### `allowAsImport`
+
+A boolean. If it's set to `true`, the require statements with an `import` declaration won't be reported. The default value is `false`. If you want to apply the `no-var-requires` rule, then set it to `true`.
```

Comment:
[Docs] Two thoughts here:

* We know it's a `boolean` with a default value of `false` from the auto-generated code block earlier in the file.
* Soon, `no-var-requires` will be a distant memory. And these rules have a lot of overlap anyway. So IMO there's no need to explicitly mention it.

```suggestion
If it's set to `true`, the require statements with an `import` declaration won't be reported.
This is useful if you're on an older CommonJS-style codebase and haven't been able to transition to ESM-style imports yet.
```